pkg/maintenance/alertManagerSilenceClient.go

```diff
@@ -9,12 +9,19 @@ import (
 	"net/http"
 )
 
+//go:generate mockgen -destination=alertManagerSilenceMock.go -package=maintenance github.com/openshift/managed-upgrade-operator/pkg/maintenance AlertManagerSilencer
```

Comment:
If I recall, was the reason for not putting it the 'mocks' package dir due to the cyclic import that results in?

Reply:
correct